src/com/tictacgo/data/Space.java

```diff
@@ -0,0 +1,100 @@
+package com.tictacgo.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Space {
+  /**
+   * A List of the Pieces which are in this Space.
+   */
+  private List<Piece> pieces;
+
+  /**
+   * Creates a new Space, initially empty.
+   */
+  public Space() {
+    pieces = new ArrayList<Piece>(2);
+  }
+
+  /**
+   * Adds a Piece into this Space.
+   *
+   * @param piece The Piece to add.
+   */
+  public void addPiece(Piece piece) {
+    pieces.add(piece);
```

Comment:
This will silently succeed if you add more than 2 pieces. IMO we should check the size of pieces and throw an exception if it's >= 2 already.